fybrik/README_mysql.md

```diff
@@ -0,0 +1,185 @@
+### Version compatibility matrix
+
+| Fybrik           | AFM     | Command
```

Comment:
For compatibility reasons all table rows should have borders (pipe symbols) at the start and at the end

The fix

| Fybrik | AFM    | Command                                                                         |
|--------|--------|---------------------------------------------------------------------------------|
| 0.7.x  | 0.1.x  | `https://github.com/fybrik/airbyte-module/releases/download/v0.1.0/module.yaml` |
| 1.0.x  | 0.2.x  | `https://github.com/fybrik/airbyte-module/releases/download/v0.2.0/module.yaml` |
| 1.1.x  | 0.2.x  | `https://github.com/fybrik/airbyte-module/releases/download/v0.2.0/module.yaml` |
| master | master | `https://raw.githubusercontent.com/fybrik/airbyte-module/master/module.yaml`    |

Reply:
OK thanks. I updated the matrix in `README.md` and removed it from `README_mysql.md`